PersistenceContentProviderSample/app/build.gradle

```diff
@@ -47,17 +47,17 @@ dependencies {
 
     // Test
     testCompile 'junit:junit:4.12'
-    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
+    androidTestCompile('com.android.support.test.espresso:espresso-core:3.0.1', {
```

Comment:
change `compile` to `Implementation` everywhere

Reply:
`PersistenceContentProviderSample` use stable Android plugin (2.3.3), so `implementation` is not available.
Should I change all projects to beta Android plugin (3.0.0-beta3)?

Reply:
@florina-muntenescu Any plan to merge?